pkg/cmd/pulumi/package_publish.go

```diff
@@ -143,11 +143,13 @@ func publishToNPM(path string) error {
 	logging.V(1).Infof("Running %s", infoCmd)
 	output, err := infoCmd.Output()
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to run npm info to verify package %w", err)
```

Comment:
@abhinav can probably find the links he always posts for this, but my understanding is we're trying to refrain from writing errors of the form "x failed" now. This should probably just be something like "run npm info to verify package: %w".

Reply:
right! update that we actually want to ignore this error altogether, as we are using stdout being `""` to verify package status.
We have been [doing this for a long time](https://github.com/pulumi/scripts/blob/master/ci/publish-tfgen-package#L86).